_posts/2016-08-04-six-months.md

```diff
@@ -18,26 +18,26 @@ Looking just at raw numbers for those first six months, that’s:
 * 761 pull requests from 45 contributors.
 * 498 Jira issues opened and 245 resolved.
 * 1 incubating release (and another 1 in progress). 
-* 4200 hours of automated tests. 
+* 4,200 hours of automated tests. 
 * 161 subscribers / 606 messages on user@.
 * 217 subscribers / 1205 messages on dev@.
 * 277 stars and 174 forks on GitHub.
 
-But behind those numbers, there’s been a ton of technical progress, including:
+And behind those numbers, there’s been a ton of technical progress, including:
 
 * Refactoring of the entire codebase, examples, and tests to be truly runner-independent.
 * New functionality in the Apache Flink runner for timestamps/windows in batch and bounded sources and side inputs in streaming mode.
 * Work in progress to upgrade the Apache Spark runner to use Spark 2.0.
 * Several new runners from the wider Apache community -- Apache Gearpump has its own feature branch, Apache Apex has a PR, and conversations are starting on Apache Storm and others.
-* New SDKs/DSLs -- the Python SDK from Google is in, and there are plans to add the Scio DSL from Spotify.
-* Support for new IO connectors -- Apache Kafka and JMS are in, with Amazon Kinesis, Apache Cassandra, and MongoDB in PR, and more in planning.
+* New SDKs/DSLs for exposing the Beam model -- the Python SDK from Google is in on a feature branch, and there are plans to add the Scio DSL from Spotify.
+* Support for additional data sources and sinks -- Apache Kafka and JMS are in, there are PRs for Amazon Kinesis, Apache Cassandra, and MongoDB, and more connectors are being planned.
```

Comment:
Sorry these didn't get addressed. I decided to limit the links in the post itself -- every line would have linked to something. Felt like it was better for the final few links to pop.